staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -204,9 +203,15 @@ func (s *ServerRunOptions) Validate() []error {
 	if err := validateCorsAllowedOriginList(s.CorsAllowedOriginList); err != nil {
 		errors = append(errors, err)
 	}
+	if s.FeatureGate == nil {
+		return errors
+	}
 	if errs := s.FeatureGate.Validate(); len(errs) != 0 {
 		errors = append(errors, errs...)
 	}
```

Comment:
`if s.FeatureGate != nil { if errs :=s.FeatureGate.Validate()... }` to accumulate all possible errors?

Reply:
done